### ISSUE TITLE
NTR: Thallium high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8438]
New term request Thallium high in blood (segal_220303230845):   UMLS: 0238452

**Comment by pnrobinson** [comment]
@MickeySegal
just to confirm that this is thallium toxicity and not an increased level of something that is normally there?

**Comment by MickeySegal** [comment]
Correct.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000445
- label: Elevated thallium cadmiun concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: PMID:30020612
- def: The concentration of thallium in the blood is above the upper limit of a guidance value for total blood mercury concentrations that implies possible injurious effects.
- comment: Thallium is a tasteless, odorless, and water-soluble chemical element for which both accidental intoxication and criminal poisoning have been reported.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8438
- synonyms:
